internal/pkg/describe/backend_service_test.go

```diff
@@ -516,7 +516,7 @@ func TestBackendServiceDescriber_Describe(t *testing.T) {
 						OperatingSystem: "LINUX",
 						Architecture:    "X86_64",
 					}, nil),
-					m.ecsDescriber.EXPECT().RollbackAlarmNames().Return(nil, nil),
+					m.ecsDescriber.EXPECT().RollbackAlarmNames().Return([]string{}, nil),
```

Comment:
+1 maybe we should fix `AlarmDescriptions`

Reply:
> why did these need to change? i think it's pretty normal to return nil instead of an empty slice...

I had changed it bc that was the cause of tests passing previously when they shouldn't have-- `RollbackAlarmNames()` was returning an empty slice, so the unit test met the condition of `alarmNames` being nil or not, but in reality it was only ever an empty slice or a non-empty slice. 

I'll change `RollbackAlarmNames` to return nil, rather than changing the condition.